pygenomeworks/genomeworks/cudaaligner/cudaaligner.pyx

```diff
@@ -122,7 +122,6 @@ cdef class CudaAlignerBatch:
     """Python API for CUDA-accelerated sequence to sequence alignment.
     """
     cdef unique_ptr[cudaaligner.Aligner] aligner
-    cdef public object stream
```

Comment:
why was this removed? I think there's a reference to this below?

Reply:
Because I would like this class to use the python object, not the c++ object.

Reply:
Does the `self.stream = stream` line in `__cinit__` work if the `stream` variable is not `cdef`ed with some type?

Reply:
I removed all the python changes from this PR.